sketch.js

```diff
@@ -38,199 +40,232 @@ var gameOver = false;
 
 var timer = 0;
 
+var isBonus = 0;
+
 function preload() {
-  /**
-   * Load Assets: images, sounds, gifs
-   */
-  shipPNG = loadImage("./assets/art/PNG/playerShip1_red.png");
-  shipDistroyedPNG = loadImage("./assets/art/PNG/Damage/playerShip1_damage3.png")
-  invaderPNG = loadImage("./assets/art/PNG/Enemies/enemyBlue2.png");
-  starsPNG = loadImage("./assets/stars.jpg");
-  starsPNG_1080p = loadImage("./assets/stars_1080p.jpg");
-  meteorPNG = loadImage("./assets/art/PNG/Meteors/meteorBrown_big2.png");
-
-  explosionGIF = loadImage("./assets/effects/explosion.gif");
-
-  soundFormats('mp3', 'ogg');
-  explosionSound = loadSound('./assets/sounds/explosion.mp3');
-  laserSound = loadSound('./assets/sounds/laser.mp3');
-  laserSound2 = loadSound('./assets/art/Bonus/sfx_laser1.ogg');
-  music = loadSound('./assets/sounds/music.mp3');
+    /**
+     * Load Assets: images, sounds, gifs
+     */
+    shipPNG = loadImage("./assets/art/PNG/playerShip1_red.png");
+    shipDistroyedPNG = loadImage("./assets/art/PNG/Damage/playerShip1_damage3.png")
+    invaderPNG = loadImage("./assets/art/PNG/Enemies/enemyBlue2.png");
+    starsPNG = loadImage("./assets/stars.jpg");
+    starsPNG_1080p = loadImage("./assets/stars_1080p.jpg");
+    meteorPNG = loadImage("./assets/art/PNG/Meteors/meteorBrown_big2.png");
+    bulletGreen = loadImage("./assets/art/PNG/Lasers/laserGreen02.png")
```

Comment:
Lets define more bulletColors here and pass them as a parameter to the Bullet Class